backend/app.ts

```diff
@@ -1,31 +1,16 @@
 import express from 'express';
-import admin from 'firebase-admin';
-// import cors from 'cors';
-import 'firebase/auth';
-import * as dotenv from 'dotenv';
+import cors from 'cors';
+import { config } from 'dotenv';
```

Comment:
hi i just wanted to know what the difference between this vs what was there before  (importing dotenv and using dotenv.config() )

Reply:
they are identical in terms of functionality!